src/components/common/ShareStructure/ShareStructures.vue

```diff
@@ -84,15 +86,19 @@ export default class ShareStructures extends Mixins(CommonMixin) {
     return (this.getNewResolutionDates.length === 0) && this.isAlterationFiling
   }
 
-  /** Keep the store in sync with components state of validity. */
-  @Watch('isEditing')
+  /**
+   * Keep the store in sync with components state of validity.
+   * Run this "immediately" to set initial state.
+   */
+  @Watch('isEditing', { immediate: true })
   @Watch('hasMinimumShareClass')
   private setShareStructureValidity (): void {
     // Check valid conditions
     const isValid = !this.isEditing && this.hasMinimumShareClass
 
     this.setEditingShareStructure(this.isEditing)
     this.setValidComponent({ key: 'isValidShareStructure', value: isValid })
+    this.setCreateShareStructureStepValidity(isValid)
```

Comment:
So here are the 2 validity architectures being used together, because different filings check validity using different methods.